nuntium/tests/moderation_messages_test.py

```diff
@@ -212,6 +212,53 @@ def test_message_that_has_not_been_confirmed_cannot_be_moderated(self):
         outbound_message_to_pedro = OutboundMessage.objects.get(message=message)
         self.assertEquals(outbound_message_to_pedro.status, 'new')
 
+    def test_error_if_moderate_public_message_created_when_moderation_off(self):
+        self.writeitinstance1.config.moderation_needed_in_all_messages = False
+        self.writeitinstance1.config.save()
+        message = Message.objects.create(
+            content='Content 1',
+            author_name='Felipe',
+            author_email="falvarez@votainteligente.cl",
+            subject='Fiera es una perra feroz',
+            public=True,
+            writeitinstance=self.writeitinstance1,
+            persons=[self.person1],
+            )
+
+        message.recently_confirmated()
+
+        with self.assertRaises(Moderation.DoesNotExist):
+            message.moderate()
+
+        self.assertFalse(message.moderated)
+        outbound_message_to_pedro = OutboundMessage.objects.get(message=message)
+        self.assertEquals(outbound_message_to_pedro.status, 'ready')
+
+    # for https://github.com/ciudadanointeligente/write-it/issues/704
+    def test_moderation_created_for_old_messages_after_moderation_turned_on(self):
+        # this message has not been confirmed
```

Comment:
"has not been confirmed" seems misleading here, since in the next statement you set it to be confirmed - maybe I'd add a "now set it as confirmed" comment afterwards.